src/main/java/medicaldoctor/controllers/UserLookupController.java

```diff
@@ -100,6 +100,7 @@ public void initialize(URL location, ResourceBundle resources) {
         
         table.setRowFactory(tv -> {
             TableRow<User> row = new TableRow<>();
+            
             row.setOnMouseClicked(event -> {
```

Comment:
I tried making your suggestion with the throws Exception but you can't overload the init method .

Reply:
you dont overload, you just add the throws declaration